pyvista/plotting/plotter.py

```diff
@@ -104,7 +105,7 @@
         KILL_DISPLAY = False
 
 
-def close_all():
+def close_all() -> bool:
```

Comment:
Weirdly, this function ONLY returns True.  But the way the documentation is written makes it seem like the intent is that it might return False too.  Let's keep it with the type hint you have to avoid any surprises later.